src/Symfony/Bundle/FrameworkBundle/Resources/config/console.xml

```diff
@@ -66,6 +66,8 @@
         </service>
 
         <service id="console.command.debug_autowiring" class="Symfony\Bundle\FrameworkBundle\Command\DebugAutowiringCommand">
+            <argument />
```

Comment:
`<argument>null</argument>`

Reply:
Fixed tests pass now, thank you for your help!